docs/source/tasks/enable-fabricv2-capability.rst

```diff
@@ -0,0 +1,113 @@
+..
+.. SPDX-License-Identifier: Apache-2.0
+..
+
+Enable Hyperledger Fabric V2 Capability
+=======================================
+
+Fabric V2 capability is distinct from Favric V2.x binaries, and the most common scenario will be where the binaries on a network have been upgraded to v2.x, leaving the Capability of the channels at V1.x.  This playbook example can be used to upgrade firstly the system channel, then an array of named application channels.
```

Comment:
Typo for `Favric` - also, can we make the capitalisation consistent with `V` and `v` for versions?

Reply:
Typos corrected.